atlp-rwanda/rca-barefoot-team4-bn
# Fix githubactions issues

What does this PR do?
=> This PR contains an updated main.yml file which runs github actions
Description of Task to be completed
=> task to complete was creating github actions badge in readme file which kept showing failure on github actions so this PR is to check of build is successfully running and github actions will succeed.
How should this be manually tested?
=> go to actions, check the workflow for build and test 
Any background context you want to provide?
=> The github actions could fail when the commands being run are not succeeding. This is because the use of those actions is to check the project activities that are running successfully and which activities are failing
What are the relevant pivot tracker story?
=> This is not connected to a pivot tracker story however it can be related to integrating github actions with readme badge
Screenshots (if appropriate) no screenshots available
